src/PhpEnvironment/Request.php

```diff
@@ -93,6 +96,9 @@ public function getContent()
     {
         if (empty($this->content)) {
             $requestBody = file_get_contents('php://input');
+            if (false === $requestBody) {
+                throw new RuntimeException('Unable to get contents from php://input');
+            }
```

Comment:
This is new behavior and requires a corresponding test.

Reply:
Fixed and replaced with simpler condition